packages/www/Dockerfile

```diff
@@ -1,12 +1,13 @@
-# To be run from the root of the repository so that we can pick up other packages
+FROM	node:14
 
-FROM node:14
+WORKDIR	/app
+
+COPY	package.json yarn.lock lerna.json	./
+
+COPY	.	.
+
```

Comment:
Not a change here, but shouldn't the `yarn install` below go between these 2 copies? Otherwise there's no point in copying the `yarn` files first AFAIK

Reply:
that's what i thought too. but wasn't sure how lerna packages work here...